#goal definition
bool stopOnBlock
float32 width
float32 speed
---
#result definition
int32 status_code
---
#feedback
int32 status_code
